serverlessworkflow/modules/ROOT/pages/cloud/operator/configuring-knative-eventing-resources.adoc

```diff
@@ -1,60 +1,425 @@
   flow:
-    events: <2>
-      - name: requestQuote
-        type: kogito.sw.request.quote
-        kind: produced
-      - name: aggregatedQuotesResponse,
-        type: kogito.loanbroker.aggregated.quotes.response,
-        kind: consumed,
-        source: /kogito/serverless/loanbroker/aggregator
-...
---
-<1> `spec.sink.ref` defines the sink that all created sinkBinding will use as the destination sink for producing events
-<2> `spec.flow.events` lists all the events referenced in the workflow. Events with `produced` kind will trigger the creation of `SinkBindings` by the {operator_name}, while those labeled as `consumed` will lead to the generation of `Triggers`.
+    start: ExampleState
+    events:
+      - name: exampleConsumedEvent1
+        source: ''
+        type: example_event_1 <3>
+        kind: consumed
+      - name: exampleConsumedEvent2
+        source: ''
+        type: example_event_2 <4>
+        kind: consumed
+----
+
+<1> Fixed env variable name that contains the URL of the broker created in <<_configuring_the_knative_broker>>.
+<2> Must contain the broker URL. To get this value <<querying_broker_url, see>>. The remaining env variables are fixed configurations, and you must add them as is.
+<3> Every consumed event requires a trigger, <<trigger-event-type1, see>>.
+<4> Every consumed event requires a trigger, <<trigger-event-type2, see>>.
+
+=== Configuring the Workflow Knative Eventing Resources
+
+For every event type consumed by the workflow you must create a corresponding trigger to deliver it from the broker.
 
 [NOTE]
 ====
-Knative resources are not watched by the operator, indicating they will not undergo automatic reconciliation. This grants users the freedom to make updates at their preference.
+Unlike the triggers related to the Data Index Service and the Jobs Service, these triggers must be created for every workflow that consume events.
+So it's recommended that you use trigger names that are linked to the workflow name.
 ====
 
+[[trigger-event-type1]]
+.Trigger to consume events of type example_event_1
+[source,yaml]
+----
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: example-workflow-example-event-1-trigger <1>
+spec:
+  broker: default
+  filter:
+    attributes:
+      type: example_event_1 <2>
+  subscriber:
+    ref:
+      apiVersion: v1
+      kind: Service
+      name: example-workflow
+      namespace: example-namespace
+----
+
+<1> Name for the trigger.
+<2> Event type consumed by the workflow `example-workflow`.
+
+[[trigger-event-type2]]
+.Trigger to consume events of type example_event_2
+[source,yaml]
+----
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: example-workflow-example-event-2-trigger <1>
+spec:
+  broker: default
+  filter:
+    attributes:
+      type: example_event_2
+  subscriber:
+    ref:
+      apiVersion: v1
+      kind: Service
+      name: example-workflow
+      namespace: example-namespace
+----
+
+:sectnums!:
```

Comment:
Can we have a brief section explaining how to verify that the setup is working? Maybe pointing to another doc that shows that?

Reply:
we can try to add something, but we are running out-of-time at the same time. I think, we should merge this, try to finish the revisting of the supporting services guide, and hopefully, add more stuff on this.